src/c++/perf_analyzer/genai-pa/tests/test_llm_metrics.py

```diff
@@ -51,19 +51,22 @@ def prepare_triton_profile_data(self) -> None:
                         {
                             "timestamp": 1,
                             "response_timestamps": [3, 5, 8],
+                            # LLMProfileDataParser preprocessse the responses
+                            # from triton server and removes first few chars.
+                            # Add whitespace to avoid valid chars being removed.
                             "response_outputs": [
-                                {"text_output": "dogs"},
-                                {"text_output": "are"},
-                                {"text_output": "cool"},
+                                {"text_output": "   dogs"},
```

Comment:
Can you add a todo to remove the white space after we resolve the binary data in a string issue in PA?

Reply:
added fixme

Reply:
thank you